.eslintrc.js

```diff
@@ -107,7 +107,7 @@ module.exports = {
     "no-undef": 2,
     "no-undefined": 2,
     "no-unused-vars": 2,
-    "no-use-before-define": 2,
+    "no-use-before-define": [2],
```

Comment:
This change can be reverted as its not required with your final revision of this pull request (no longer triggering use before define error)